ACL/acl-schema.sql

```diff
@@ -0,0 +1,97 @@
+create database if not exists acl;
+use acl;
+ 
+#Hence user type is new user(root user) or administrater
+CREATE TABLE IF NOT EXISTS userType(
+    type_id int,
+    user_type varchar(15),
+    PRIMARY KEY(user_type)
+);
+
+insert into userType values(101,'User');
+insert into userType values(102,'Admin');
+
+
+#here we create a user table for any user type to store information
+CREATE TABLE IF NOT EXISTS user(
+    user_id int,
+    user_name varchar(30),
+    password varchar(10),
+    user_type varchar(15),
```

Comment:
One use can have two role, e.g. 
`user 1` can be admin for `directory-1` but having read only access to `directory-2`

How you handle this case?